content/livesync/index.textile

```diff
@@ -1,6 +1,57 @@
+</a>
+
+- User interaction := with the application's UI involves entering data stored in the local state.
+- Optimistic update := means the update immediately reflects the user's changes in the UI.
+- Local State := changes made by the user are first written to the application's local state, which then sends these changes to the server.
+- Server API := sends a write request to the server, which processes this request through an API endpoint. The server then attempts to write the changes in a transactional form to the database.
+- The database's := new state is saved in database tables.
+- A confirmed State := is then returned to the application from the server, signaling that the changes have been successfully persisted.
+- "Ably Realtime Data Models SDK":https://https://github.com/ably-labs/models := includes tools and libraries to handle the data models and synchronize with the server.
+- Subscription to change := allows the app to receive updates about confirmed state in realtime
+
+h2. LiveSync’s Key Capabilities
+
+* *Realtime database synchronisation* integrates clients with relational databases in realtime. The primary reason for using LiveSync is it directly impacts the user experience by providing immediate data updates between multiple clients.
+
+* *Data integrity and delivery guarantees* guarantees data integrity and the exact-once delivery of synchronised data changes and user actions.
+
+* *Stateful interactive applications* to maintain consistency across multiple clients effortlessly.
+
+* *Optimistic updates at  API Level* makes updates at the API level, LiveSync offers users an immediate view of changes, eliminating the wait for backend confirmation.
+
+* *Scalability and reliability* means LiveSync manages large user spikes and interactive actions reliably, supported by a robust uptime service-level agreement (SLA).
+
+* *LiveSync alerts subscribers* to new data, eliminating server polling.
+
+* *Integration with all databases* designed to work with any type of database you prefer.
+
+h2. How LiveSync works
+
+When a change occurs in the database, such as adding a new document, updating an existing document or deleting a document, LiveSync detects it immediately. It then sends this change to all connected clients in realtime, ensuring that every user sees the latest state of data without having to refresh or query the database manually.
+
+This diagram illustrates how LiveSync works:
+
+<a href="/images/diagrams/live-sync-models-SDK.png" target="_blank">
+  <img src="/images/diagrams/live-sync-models-SDK.png" style="width: 100%" alt="Basic authentication process diagram">
+</a>
+
+
+- Sync function := initialises the application state by loading the most current data from the backend, ensuring the client begins with an up-to-date model.
+- Mutation function := captures any modifications made to the client's model and communicates these changes to the server for synchronisation and persistence.
```

Comment:
We no longer have mutation functions - these are done outside of the SDK. Users do define merge functions though which define how to update the state when a change event is received